code/modules/mob/living/carbon/human/species_types/psyphoza.dm

```diff
@@ -66,10 +66,26 @@
 	PH?.Trigger()
 
 /datum/species/psyphoza/get_species_description()
-	return "..."
+	return "Psyphoza are a species of extra-sensory lesser-sensory fungal, \
+	based humanoids, infamous for their invulnerability to occlusion based \
+	magic tricks."
 
 /datum/species/psyphoza/get_species_lore()
-	return list("...")
+	return list(
+		"A standing testament to the humor of mother nature, Psyphoza have evolved powerful and mystical \
+			psychic abilities, which are almost completely mitigated by the fact they are absolutely \
+			blind, and depend entirely on their psychic abilities to navigate their surroundings.",
+
+		"Psyphoza culture is deeply rooted in superstition, mysticism, and the occult. It is their belief \
+			that the morphology of their cap deeply impacts the course of their life, with characteristics \
+			such as size, colour, and shape influencing how irrespectively lucky or unlucky they might be in \
```

Comment:
Colour is not really something the Psyphoza can observe themselves, or likely even comprehend due to blindness.

Or are we actually going with the "legally blind" thing?

Reply:
Legally blind is the most appealing to me. Pinhole vision, essentially how it is in-game. Besides, not like you can't handwave some horse shit about psychic sense and colour.

Reply:
👍